src/shared/store/bottomSheetStore.ts

```
@@ -0,0 +1,26 @@
+import { create } from 'zustand';
+
+// Registry Pattern: Features will augment this interface
+// eslint-disable-next-line @typescript-eslint/no-empty-object-type
+export interface BottomSheetMap {}
```

Biome (2.1.2)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)